Dockerfile

```diff
@@ -1,4 +1,16 @@
-FROM golang:1.16-buster
+ARG MONGODB_TOOLS_VERSION=100.5.1
+ARG EN_AWS_CLI=true
+ARG AWS_CLI_VERSION=1.22.46
+ARG EN_AZURE=true
+ARG AZURE_CLI_VERSION=2.32.0
+ARG EN_GCLOUD=true
+ARG GOOGLE_CLOUD_SDK_VERSION=370.0.0
+ARG EN_GPG=true
+ARG GNUPG_VERSION="2.2.31-r0"
+ARG EN_MINIO=true
+ARG EN_RCLONE=true
+
+FROM golang:1.15 as mgob-builder
```

Comment:
Go 1.15 is deprecated it should work with Go 1.17

Reply:
👍🏼